### ISSUE TITLE
[P2][Sourcing][Accessibility] Checkbox focus is not visible when pressing spacebar on select/deselect all

### ISSUE THREAD

**Issue by CSEHoangV** [SAP/fundamental-ngx#11754]
#### Is this a bug, enhancement, or feature request?

bug

#### Describe your proposal.

On platform table, when we try to select/deselect all by pressing spacebar, most of time the screen scroll down. It does not happen when we press enter.

https://github.com/SAP/fundamental-ngx/assets/162270184/f384b0bc-8bd0-4ffd-94ce-794c0aa69c1b

#### Can you handle that on the application side

No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

Angular 15
Current version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

- Access platform table having multi selection mode enabled.
- Select/deselect by pressing spacebar.

#### Please provide relevant source code (if applicable).

https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#freezable

#### Please provide stackblitz example(s).

https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#freezable

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.

Yes. This is reported by our accessibility expert. 

#### Did you check the documentation and the API?

Yes

#### Did you search for similar issues?

Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**
